projects/1.21/assets/easters-delight/eastersdelight/lang/zh_cn.json

```
@@ -0,0 +1,30 @@
+{
+  "block.eastersdelight.egg_painter": "彩蛋画架",
+  "eastersdelight.configuration.enable_villager_trades": "开启村民交易",
+  "eastersdelight.configuration.enable_wandering_trader_trades": "开启流浪商人交易",
+  "eastersdelight.configuration.enable_dyed_eggs_in_creative_mode": "在创造模式开启彩蛋",
+  "eastersdelight.container.egg_painter": "彩蛋画架",
+  "eastersdelight.container.egg_painter.missing_base_color_tooltip": "添加基础染料",
+  "eastersdelight.container.egg_painter.missing_egg_tooltip": "添加煮鸡蛋",
+  "eastersdelight.container.egg_painter.missing_pattern_color_tooltip": "添加图案染料",
+  "eastersdelight.container.egg_painter.missing_pattern_tooltip": "添加鸡蛋图案",
+  "eastersdelight.tooltip.egg_pattern.blocks": "§7方块§r",
+  "eastersdelight.tooltip.egg_pattern.creeper": "§7苦力怕§r",
+  "eastersdelight.tooltip.egg_pattern.dipped": "§7沉浸§r",
+  "eastersdelight.tooltip.egg_pattern.dots": "§7圆点§r",
+  "eastersdelight.tooltip.egg_pattern.heart": "§7爱心§r",
+  "eastersdelight.tooltip.egg_pattern.petals": "§7花瓣§r",
+  "eastersdelight.tooltip.egg_pattern.split": "§7分裂§r",
+  "eastersdelight.tooltip.egg_pattern.stripes": "§7条纹§r",
+  "eastersdelight.tooltip.egg_pattern.stripes_2": "§7条纹II§r",
+  "eastersdelight.tooltip.egg_pattern.stripes_3": "§7条纹III§r",
```

```suggestion
  "eastersdelight.tooltip.egg_pattern.stripes_2": "§7条纹二号§r",
  "eastersdelight.tooltip.egg_pattern.stripes_3": "§7条纹三号§r",
```
参照了机械动力的翻译方法，仅作风格化处理，不强求修改